front/src/app/app.module.ts

```diff
@@ -71,8 +75,12 @@ const routes: Routes = [
     RouterModule.forRoot(routes),
     NgxsModule.forRoot([
       ScanState,
-      UserState
-    ])
+      UserState,
+      LoginState
+    ]),
+    NgxsStoragePluginModule.forRoot({
+      key: ['auth.token', 'auth.username']
```

Comment:
Dlaczego chcecie trzymać nazwę użytkownika w localstorage-u?

Reply:
Inaczej przy odświeżaniu strony znika mi to z navbaru. Myślałem, że skoro zostaje token to username też może